src/App.js

```diff
@@ -1,10 +1,25 @@
 import Home from './views/Home/Home'
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect
+} from 'react-router-dom'
 
 function App() {
   return (
-    <main className='App bg-gray-100 dark:bg-gray-800 text-gray-900 dark:text-gray-50'>
-      <Home />
-    </main>
+    <Router>
+      <main className='App bg-gray-100 dark:bg-gray-800 text-gray-900 dark:text-gray-50'>
+        <Switch>
+          <Route exact path='/'>
+            <Home />
+          </Route>
+          <Route path='*'>
```

Comment:
At the moment I just want to redirect to the root path, maybe later I can add more views